cpp/src/glm/qn/qn_util.h

```diff
@@ -147,7 +146,7 @@ inline bool check_convergence(const LBFGSParam<T> &param, const int k,
 /*
  * Multiplies a vector g with the inverse hessian approximation, i.e.
  * drt = - H * g,
- * e.g. to compute the new search direction for g = \nabla f(x)
```

Comment:
I think this is supposed to be `nabla` here, which is being escaped. This looks like Latex.

Reply:
Ah.. Thanks. Fixed this one.